mentat/parsers/file_edit.py

```diff
@@ -86,12 +86,9 @@ def is_valid(
                     color="red",
                 )
                 return False
-            elif rel_path not in code_file_manager.file_lines:
-                cprint(
-                    f"File {rel_path} not in context, canceling all edits to file.",
-                    color="red",
-                )
-                return False
+            # elif rel_path not in code_file_manager.file_lines:
```

Comment:
Is this supposed to still be commented out or just a WIP?

Reply:
WIP. There's a test for this btw and I have it marked xfail atm.